lib/image_db.rb

```diff
@@ -113,17 +123,36 @@ def full_update
       else
         url = nil
       end
-      products = data["_embedded"]["goods"].map {|g| "http://verygoods.co" + g["_links"]["product"]["href"].gsub(/products/, 'product') }
-      products.each {|p| @images.add p }
+      products = data["_embedded"]["goods"].map { |g| "http://verygoods.co" + g["_links"]["product"]["href"].gsub(/products/, "product") }
+      products.each { |p| @images.add p }
+      puts "Images: #{@images.count}"
+    end
+
+    # Instagram
+    #
+    # No idea if the max_id stuff works here, all instagram likes currently fit
+    # in one page.
+    max_id = nil
+    user = ImageDB.instagram_client.user.username
+    loop do
+      p ({ instagram: max_id, user: user })
```

Comment:
`(...)` interpreted as grouped expression.<br>Do not write to stdout. Use Rails' logger if you want to log.